openqabot/types/incidents.py

```diff
@@ -141,7 +140,10 @@ def _handle_incident(  # noqa: PLR0911,C901, PLR0917
         full_post["qem"]["version"] = self.settings["VERSION"]
         full_post["openqa"]["DISTRI"] = self.settings["DISTRI"]
         full_post["openqa"]["_ONLY_OBSOLETE_SAME_BUILD"] = "1"
+        if channel.product == "SLFO":
+            full_post["openqa"]["_OBSOLETE"] = "1"
```

Comment:
Why bother with SLFO at all? As described in https://progress.opensuse.org/issues/192436 as well I think we can just schedule all incident tests with OBSOLETE. And we should avoid any product specific code in here


```suggestion
        full_post["openqa"]["_OBSOLETE"] = "1"
```